src/components/UserInput/UserInput.js

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import './UserInput.scss';
+
+const UserInput = ({
+  name,
+  koreanName,
+  condition,
+  inputValue,
+  loginInputHandler,
+}) => {
+  const [inputTitleGoUpBtn, setInputTitleGoUpBtn] = useState(false);
+
+  const inputGoUpEffector = () => {
+    if (inputValue.length > 0) return;
+    else {
+      setInputTitleGoUpBtn(prev => !prev);
+    }
+  };
+
+  return (
+    <>
+      <div className="user-input">
+        <span
+          className={`user-placeholder 
+            ${inputTitleGoUpBtn ? 'input-title-goup' : null}
+           ${!condition && inputValue.length > 0 ? 'input-red' : 'input-gray'}`}
+        >
+          {koreanName} *
+        </span>
+        <input
+          type={name}
+          name={name}
+          className={
+            !condition && inputValue.length > 0 ? 'input-border-red' : null
+          }
+          onFocus={e => {
+            inputGoUpEffector();
+          }}
+          onBlur={e => {
+            inputGoUpEffector();
+          }}
```

Comment:
```suggestion
          onFocus={inputGoUpEffector}
          onBlur={inputGoUpEffector}
```
e를 사용하지 않는다면 이와같이 함수를 입력할 수 있습니다.